### DISCUSSION TITLE
Daily Standup - Wednesday, May 17

### DISCUSSION THREAD

**Discussion by DavBot02** [antonieto/ibm-dashboard#181] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Merged #180 

**What are your priorities for the next 24 hours?**:
 - Work with @Int-Angel in dashboard crud (or anything else)

**Blockers**:
 - None really

**Shoutouts**:
 - @antonieto for a succesful debugging session

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - I started working on the dataSources feature

**What are your priorities for the next 24 hours?**:
 - Keep woring ont the dataSources feature
 - Start working on the architecture document

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02 for the auth feature :)
